paws_collector.js

```diff
@@ -96,14 +96,14 @@ class PawsCollector extends AlAwsCollector {
         }
     }
 
-    constructor(context, {aimsCreds, pawsCreds}, childVersion, healthChecks = [], statsChecks = []) {
+    constructor(context, {aimsCreds, pawsCreds}, childVersion,streams =[],healthChecks = [], statsChecks = []) {
```

Comment:
@daniel-greening has a proposal to use `collector_stream` env var. Please see [here](https://github.com/alertlogic/paws-collector/pull/171#pullrequestreview-578447101).
I think it is worth implementing it right away.
@daniel-greening @rakhimundhada15 Thoughts?

Reply:
I can implement and As  per my understanding we have to made changes in below files of paws-collector:
[ catalog.json](https://algithub.pd.alertlogic.net/defender/applications/blob/master/priv/catalog.json#L1514)
[salesforce-collector.template](https://github.com/alertlogic/paws-collector/blob/master/collectors/salesforce/cfn/salesforce-collector.template#L113)
[collector.js](https://github.com/alertlogic/paws-collector/blob/master/collectors/salesforce/collector.js#L39) 
[sam-template](https://github.com/alertlogic/paws-collector/blob/master/collectors/salesforce/local/sam-template.yaml#L30) 
Then we don't have to pass stream array from each collector as we can use common variable in Paws-collector.js

Reply:
As discussed with Dan , I am directly using collector_stream here and rest of change in other file he will take care.